platform-api-docs/scripts/specs/orgs-workspaces-operations-overlay-1.88.yml

```diff
@@ -0,0 +1,292 @@
+  
+  # ===== WORKSPACES - OPERATIONS =====
+  
+  # ---- LIST WORKSPACES ----
+  
+  - target: "$.paths./orgs/{orgId}/workspaces.get.summary"
+    update: "List workspaces"
+  
+  - target: "$.paths./orgs/{orgId}/workspaces.get.description"
+    update: "Lists all workspaces in the organization identified by the given `orgId` to which the requesting user belongs. Workspaces are isolated environments within an organization where users can run pipelines and manage resources."
+  
+  # ---- CREATE WORKSPACE ----
+  
+  - target: "$.paths./orgs/{orgId}/workspaces.post.summary"
+    update: "Create workspace"
+  
+  - target: "$.paths./orgs/{orgId}/workspaces.post.description"
+    update: "Creates a new workspace in the organization identified by the given `orgId`. The requesting user is automatically added as an owner of the workspace."
+  
+  # ---- VALIDATE WORKSPACE NAME ----
+  
+  - target: "$.paths./orgs/{orgId}/workspaces/validate.get.summary"
+    update: "Validate workspace name"
+  
+  - target: "$.paths./orgs/{orgId}/workspaces/validate.get.description"
+    update: "Confirms that the given workspace name is valid and available for use within the organization. Append `?name=<your_workspace_name>` to specify the name to validate."
+  
+  # ---- DESCRIBE WORKSPACE ----
+  
+  - target: "$.paths./orgs/{orgId}/workspaces/{workspaceId}.get.summary"
+    update: "Describe workspace"
+  
+  - target: "$.paths./orgs/{orgId}/workspaces/{workspaceId}.get.description"
+    update: "Retrieves the details of the workspace identified by the given `workspaceId`."
+  
+  # ---- UPDATE WORKSPACE ----
+  
+  - target: "$.paths./orgs/{orgId}/workspaces/{workspaceId}.put.summary"
+    update: "Update workspace"
+  
+  - target: "$.paths./orgs/{orgId}/workspaces/{workspaceId}.put.description"
+    update: "Updates the details of the workspace identified by the given `workspaceId`."
+  
+  # ---- DELETE WORKSPACE ----
+  
+  - target: "$.paths./orgs/{orgId}/workspaces/{workspaceId}.delete.summary"
+    update: "Delete workspace"
+  
+  - target: "$.paths./orgs/{orgId}/workspaces/{workspaceId}.delete.description"
+    update: "Deletes the workspace identified by the given `workspaceId`. This operation permanently removes the workspace and all associated resources, pipelines, and data."
+  
+  # ---- LIST WORKSPACE PARTICIPANTS ----
+  
+  - target: "$.paths./orgs/{orgId}/workspaces/{workspaceId}/participants.get.summary"
+    update: "List workspace participants"
+  
+  - target: "$.paths./orgs/{orgId}/workspaces/{workspaceId}/participants.get.description"
+    update: "Lists all participants of the workspace identified by the given `workspaceId`. Participants are users or teams who have been granted access to a workspace and can be assigned roles such as owner, admin, maintain, or view."
```

Comment:
> such as owner, admin, maintain, or view.

I understand the list is meant to include only some examples and not the full list of predefined roles.